metric-anomaly-data-model/src/main/avro/MetricAnomalyNotificationEvent.avdl

```diff
@@ -5,5 +5,11 @@ protocol MetricAnomalyNotificationEventProtocol {
     string event_condition_type;
     long violation_timestamp;
     string channel_id;
+    array<MetricValues> metric_values_list;
+  }
+
+  record MetricValues {
+    double rhs;
+    array<double> lhs;
```

Comment:
if the condition is true for more than 5 mins, this may be a big array.
I think, we can just go with a message saying:
` the value was higher than threshold `12` for a min` or so.

Reply:
Can we limit the size of the array? Say only 5 values or something like that ?

Reply:
this isn't being used any longer